### PR TITLE
Providing the way how a page type can be created for a new route

### PR DESCRIPTION
**Pull request by eduard13**
## Purpose of this pull request

This pull request (PR) updates the routing documentation, by providing a way of how to create a new Page Type for a new route.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/extension-dev-guide/routing.html
- https://devdocs.magento.com/guides/v2.3/extension-dev-guide/routing.html

## Links to Magento source code

-  https://github.com/magento/magento2/blob/2.3/app/code/Magento/Cms/etc/frontend/page_types.xml#L8

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/extension-dev-guide/routing.md [hunk 1]
```diff
@@ -350,6 +350,30 @@ As a result, by accessing the `http://site.com/learning` route, the `http://site
...
+    <type id="routing_index_index" label="Routing Page"/>
+</page_types>
+```
+
+{:.bs-callout-info}
+The `routing_index_index` represents the path for the new route.
+
+### Result
+
+As result, the new page is available in `Specified Page` dropdown, on widget creating page.
```
Comment (jeff-matthews) [inline src/guides/v2.3/extension-dev-guide/routing.md]: This is a little unclear. Is this rewrite accurate?

> As result, the new page is available in the `Specified Page` dropdown widget.

File: src/guides/v2.3/extension-dev-guide/routing.md [hunk 2]
```diff
@@ -350,6 +350,30 @@ As a result, by accessing the `http://site.com/learning` route, the `http://site
 
 ![Routing Result]({{ site.baseurl }}/common/images/routing-result.png)
 
+## Declaring the new route as Page Type
+
+After creating a new route `routing/index/index`, it's a good practice to give more control on it for the admin. By creating a new `Page Type`, the admin can manage the content of this page using widgets.
```
Comment (jeff-matthews) [inline src/guides/v2.3/extension-dev-guide/routing.md]:
```suggestion
After creating a new route `routing/index/index`, it is a good practice to give admins more control over it. By creating a new `Page Type`, an admin can manage the content of this page using widgets. 
```

File: src/guides/v2.2/extension-dev-guide/routing.md [hunk 3]
```diff
@@ -345,6 +345,30 @@ As a result, by accessing the `http://site.com/learning` route, the `http://site
 
 ![Routing Result]({{ site.baseurl }}/common/images/routing-result.png)
 
+## Declaring the new route as Page Type
+
+After creating a new route `routing/index/index`, it's a good practice to give more control on it for the admin. By creating a new `Page Type`, the admin can manage the content of this page using widgets.
```
Comment (jeff-matthews) [inline src/guides/v2.2/extension-dev-guide/routing.md]:
```suggestion
After creating a new route `routing/index/index`, it is a good practice to give admins more control of it. By creating a new `Page Type`, an admin can manage the content of this page using widgets. 
```

File: src/guides/v2.2/extension-dev-guide/routing.md [hunk 4]
```diff
@@ -345,6 +345,30 @@ As a result, by accessing the `http://site.com/learning` route, the `http://site
...
+    <type id="routing_index_index" label="Routing Page"/>
+</page_types>
+```
+
+{:.bs-callout-info}
+The `routing_index_index` represents the path for the new route.
+
+### Result
+
+As result, the new page is available in `Specified Page` dropdown, on widget creating page.
```
Comment (jeff-matthews) [inline src/guides/v2.2/extension-dev-guide/routing.md]: This is a little unclear. Is this rewrite accurate?

> As result, the new page is available in the `Specified Page` dropdown widget.

File: src/guides/v2.3/extension-dev-guide/routing.md (hunk 1 above)
Comment (rogyar) [reply src/guides/v2.3/extension-dev-guide/routing.md]: Hi @jeff-matthews. I believe it is. He means the following section on the widget creation page

image

Comment (eduard13) [reply src/guides/v2.3/extension-dev-guide/routing.md]: Yes, that's right.

**Pull request merged**